Content/SandboxieTrace.md

```diff
@@ -14,37 +16,44 @@ The trace displays access attempts and makes it possible to somewhat easily iden
+*   **NetFwTrace** traces the actions of the firewall components (since version 0.9.0 / 5.51.0)
 
-Each setting accepts a sequence of characters which specifies what to log. The character _a_ logs requests which were allowed; the character _d_ logs requests which were denied. For the **FileTrace** and **PipeTrace** settings, the character _i_ logs requests which were allowed because they access a device which is ignored by Sandboxie, such as a CD ROM.
+Each setting accepts a sequence of characters which specifies what to log. The character _a_ logs requests which were allowed; the character _d_ logs requests which were denied. For the **FileTrace** and **PipeTrace** settings, the character _i_ logs requests which were allowed because they access a device which is ignored by Sandboxie, such as a CD-ROM.
 
-The last three settings, **PipeTrace**, **IpcTrace** and **GuiTrace**, are more relevant to the discussion in this page. **FileTrace** and __KeyTrace_ will usually not be able to provide insight as to why a sandboxed program is malfunctioning.
+The settings **PipeTrace**, **IpcTrace** and **GuiTrace** are more relevant to the discussion in this page. **FileTrace** and **KeyTrace** will usually not be able to provide insight as to why a sandboxed program is malfunctioning.
 
-Thus, typically you enable by the trace by making this change in SandboxieIni:
+Thus, typically you enable the trace by making this change in [Sandboxie Ini](SandboxieIni.md):
 ```
    [GlobalSettings]
    IpcTrace=ad
    PipeTrace=ad
    GuiTrace=ad
 ```
 
-Then use Sandboxie Control to reload the configuration.
+Then use Sandboxie to reload the configuration:
+* **Configure** menu -> **Reload Configuration** on Sandboxie Classic
+* **Options** menu -> **Reload ini file** on Sandboxie Plus
+
+Trace options can be set on a per box basis such that only the boxes you need will generate trace logs.
 
-### Review the Trace for **IpcTrace** and **PipeTrace**
+You can also adjust the buffer size by adding ```TraceBufferPages=2560``` that will increase it tenfold.
 
-The information collected by the Sandboxie trace facility is logged in the system debugger log. You will need a utility to display this log: [DebugView](https://docs.microsoft.com/it-it/sysinternals/downloads/debugview) from Sysinternals (now Microsoft) is recommended.
+### Review the Trace for **NetFwTrace**, **IpcTrace** and **PipeTrace**
 
-(On Windows Vista, output from the system debugger log is disabled by default. [This blog post](https://web.archive.org/web/20080731211018/http://blogs.msdn.com:80/doronh/archive/2006/11/14/where-did-my-debug-output-go-in-vista.aspx) explains how to enable the output.)
+Since version 0.9.0 / 5.51.0, it was added a new option `NetFwTrace=*` to trace the actions of the firewall components. Please note that the driver only logs the kernel debug output, hence you can use [DbgView.exe](https://docs.microsoft.com/it-it/sysinternals/downloads/debugview).

```

Comment:
```suggestion
Since version 0.9.0 / 5.51.0, a new option `NetFwTrace=*` was added to trace the actions of the firewall components. Please note that the driver only logs to the kernel debug output, which you can view with [DbgView.exe](https://docs.microsoft.com/en-us/sysinternals/downloads/debugview).

```

Reply:
changed link to the english site